src/components/visualizations/chart/test/chartOptionsBuilder.spec.ts

```diff
@@ -3546,7 +3597,7 @@ describe("chartOptionsBuilder", () => {
                         {
                             data: [
                                 { x: 0, y: 0, value: 21978695.46, drilldown: false },
-                                { x: 1, y: 0, value: 6038400.96, drilldown: false },
+                                { x: 1, y: 0, value: 60038400.96, drilldown: false },
```

Comment:
why does it change?

Reply:
This changes is for reason. Current mock data is sorted and change value to make data is un-sorted then the test can check that data is sorted.